CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## 3.0.0 (2021-03-??)
+
+- feat: merge `next-on-netlify` into `@netlify/plugin-nextjs`
+  
+  This improves our ability to maintain the plugin, reduces confusion about how to configure Next.js support for Netlify builds, and prepares the plugin for something we're pretty dang excited about: automatic installation for new Next.js projects!
+
+> **NOTE: There are no breaking changes for existing plugin users.** This is a major release because we're bringing an external package into the plugin _and_ turning on automatic installation for new Next.js projects on Netlify. This doesn't break anything, but it's a big enough change that we wanted to treat it as a major release.
+
+### Migrating from v2 to v3
+
+If you've installed this plugin via Netlify's UI, no action is necessary. The upgrade is automatic.
+
+If you've installed this plugin manually in your `netlify.toml`, you can upgrade two ways:
+
+1. Use the new automatic install! Remove this plugin from your project and `netlify.toml` and Netlify will automatically install it for a seamless experience working with Next.js on Netlify.
```

Comment:
is this facts? i was under the impression this will only happen for *new* sites cc @verythorough could be v wrong though or could be a v outdated assumption 😅